lib/bulk/common.js

```diff
@@ -390,19 +402,19 @@ var mergeBatchResults = function(ordered, batch, bulkResult, err, result) {
   if (result.writeConcernError) {
     bulkResult.writeConcernErrors.push(new WriteConcernError(result.writeConcernError));
   }
-};
+}
 
 //
 // Clone the options
-var cloneOptions = function(options) {
-  var clone = {};
-  var keys = Object.keys(options);
-  for (var i = 0; i < keys.length; i++) {
+function cloneOptions(options) {
```

Comment:
do we use this in a functionally different way than `Object.assign({}, options)` ?